lib/anycable/config.rb

```diff
@@ -29,5 +47,31 @@ def initialize(*)
     def http_health_port_provided?
       !http_health_port.nil? && http_health_port != ""
     end
+
+    # Build gRPC server parameters
+    def to_grpc_params
+      {
+        pool_size: rpc_pool_size,
+        max_waiting_requests: rpc_max_waiting_requests,
+        poll_period: rpc_poll_period,
+        pool_keep_alive: rpc_pool_keep_alive,
+        server_args: rpc_server_args
+      }
+    end
+
+    # Build Redis parameters
+    def to_redis_params
+      { url: redis_url }.tap do |params|
+        params[:sentinels] = redis_sentinels unless redis_sentinels.empty?
+      end
+    end
+
+    # Build HTTP health server parameters
+    def to_http_health_params
```

Comment:
Why using `to_` prefix it these methods' names? They do not look like you casting anything (to_i, to_s). Instead, you're just fetching some subset of configuration values.

Reply:
Makes sense.

Feel free to propose a PR removing all these `to_` prefixes)